specification/dimensions/adjustmenttype.md

```diff
@@ -0,0 +1,54 @@
+# adjustment type
+
+Adjustment Type indicates the record represents a change to your monthly invoice through after-the-fact adustements (e.g. refunds, pricing changes). An adjustment can be positive or negative.
+Adjustment Type can be used by practitioners to assess any impacts to spend from positive or negative adjustments.
```

Comment:
Not all organizations have monthly invoices. But given this happens before the invoice, I'm wondering if we need that explicit reference at all.

Should we start with an intro definition to what an "adjustment" is? I pulled in the definition from ChargeType.

Can we be a little more precise with the use case? I feel like there's something better but will suggest something as a starting point. (I also aligned it to how we usually phrase the scenarios.)
```suggestion
An adjustment is a positive or negative change in cost applied after the original usage or purchase record(s). Adjustments may be related to one or more charges and are identified by the ChargeType column.

Adjustment Type indicates what kind of after-the-fact adjustment the record represents. Adjustment Type is commonly used to identify changes like credits and refunds.
```